tests/test_modeling_common.py

```diff
@@ -587,6 +587,59 @@ def test_inputs_embeds(self):
             with torch.no_grad():
                 model(**inputs_dict)
 
+    def test_lm_head_model_random_generate(self):
```

Comment:
@LysandreJik would that logic work for the fast random language generation testing?

Reply:
Yes, I like that a lot! I think you nailed it.